doc/en/cache.rst

```diff
@@ -35,13 +35,16 @@ Rerunning only failures or failures first
 
 First, let's create 50 test invocation of which only 2 fail::
```

Comment:
If one adds a code-block, we need to remove the implicit code block (`::`) from the previous line, that's why the build is failing.